control-plane/pkg/reconciler/trigger/trigger_test.go

```diff
@@ -1060,7 +1060,9 @@ func triggerReconciliation(t *testing.T, format string, env config.Env, useNewFi
 						},
 					},
 				}, env.DataPlaneConfigMapNamespace, env.ContractConfigMapName, env.ContractConfigMapFormat),
-				BrokerDispatcherPod(env.SystemNamespace, nil),
+				BrokerDispatcherPod(env.SystemNamespace, map[string]string{
+					base.VolumeGenerationAnnotationKey: "0",
+				}),
```

Comment:
This is because the test wants to test "no operations" and without an annotation now the control plane will set the annotation on the data plane so that the CM and the annotation is fully consistent